doc/source/user/basics.types.rst

```diff
@@ -291,7 +291,7 @@ or when it checks specifically whether a value is a Python scalar. Generally,
 problems are easily fixed by explicitly converting array scalars
 to Python scalars, using the corresponding Python type function
 (e.g., :class:`int`, :class:`float`, :class:`complex`, :class:`str`,
-:class:`unicode`).
+``unicode`` (v2.x)).
```

Comment:
Python2 is long gone, I think we can remove this.

Reply:
Thanks, removed it.